tests/testwebsockets.nim

```diff
@@ -456,6 +456,23 @@ suite "Test Closing":
     await session.close()
     check session.readyState == ReadyState.Closed
 
+  test "Mutual closing":
+    proc handle(request: HttpRequest) {.async.} =
+      check request.uri.path == WSPath
+      let server = WSServer.new(protos = ["proto"])
```

Comment:
FYI `protos` is optional now, many of old test cases are misleading examples. they actually need no protos  for majority of test cases.